src/Components/Categories/CategoriesTable.js

```diff
@@ -0,0 +1,144 @@
+import { faEdit } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Modal from "./../Elements/Modal";
+import CategoryEditForm from "./../Categories/CategoryEditForm";
+import {
+  updateCategory,
+  deleteCategory,
+} from "../../Services/Organizations/CategoriesService";
+import { useCurrentOrganizationContext } from "../../Contexts/currentOrganizationContext";
+
+library.add(faTrashAlt);
+library.add(faEdit);
+
+export default function CategoriesTable(props) {
+  const renderDateColumn = (dateString) =>
+    formatDate(parseISO(dateString), "PP");
+
+  const [name, setName] = useState("");
+  const [selectedCategory, setSelectedCategory] = useState({});
+
+  const [showCategoryEdit, setShowCategoryEdit] = useState(false);
+  const handleClose = () => {
+    setShowCategoryEdit(false);
+  };
+  const {
+    currentOrganizationStore,
+    currentOrganizationDispatch,
+    organizationClient,
+  } = useCurrentOrganizationContext();
+  const currentOrganizationId =
+    currentOrganizationStore.currentOrganization &&
+    currentOrganizationStore.currentOrganization.id;
+
+  const handleShowCategoryEdit = (selectedCategory) => {
+    setSelectedCategory(selectedCategory);
+    setShowCategoryEdit(true);
+  };
+
+  const handleSubmitEditCategory = ({ newName }, id) => {
+    console.log(id);
+    updateCategory(organizationClient, id, {
+      name: newName,
+      organization_id: currentOrganizationId,
+    })
+      .then((category) => {
+        setName(name);
+        handleClose();
+      })
+      .catch((error) => {
+        console.log("category update error", error);
+      });
+  };
+
+  const handleCancel = (event) => {
+    handleClose();
+  };
+
+  const handleCategoryDelete = (categoryId) => {
+    console.log("deleted!");
+  };
+
+  const columns = useMemo(
+    () => [
+      { Header: "Name", accessor: "name" },
+      {
+        Header: "Date Created",
+        accessor: (row) => renderDateColumn(row.created_at),
+      },
+      {
+        Header: "Last Modified",
+        accessor: (row) => renderDateColumn(row.updated_at),
+      },
+    ],
+    []
+  );
+
+  const categories = useMemo(() => props.categories, [props.categories]);
+  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } =
+    useTable({ columns, data: categories });
+
+  const header = headerGroups.map((headerGroup) => (
+    <tr {...headerGroup.getHeaderGroupProps()}>
+      {headerGroup.headers.map((column) => (
+        <th {...column.getHeaderProps()}>{column.render("Header")}</th>
+      ))}
+    </tr>
+  ));
+
+  const body = rows.map((row) => {
+    prepareRow(row);
+
+    return (
+      <tr {...row.getRowProps()}>
+        {row.cells.map((cell) => {
+          const renderedCell = (() => {
+            return cell.render("Cell");
+          })();
+
+          return <td {...cell.getCellProps()}>{renderedCell}</td>;
```

Comment:
I'm thinking we can shorten some of the extra stuff here since we don't need to write any complex rendering based on logic.

```suggestion
          return <td {...cell.getCellProps()}>{cell.render("Cell")}</td>;
```

I used this IIFE with the other table because I had to do highlighting based certain conditions and didn't really want to write a bunch of nested ternaries to accomplish that.